wls-wahlvorbereitung-service/src/main/resources/db/migrations/oracle/V4_0__createTableEroeffnungsuhrzeit.sql

```diff
@@ -0,0 +1,6 @@
+create table Eroeffnungsuhrzeit
+(
+    wahlbezirkid          varchar(1000) not null,
+    eroeffnungsuhrzeit TIMESTAMP     not null,
+    primary key (wahlbezirkid)
```

Comment:
Hat IntelliJ bei dir nicht formartiert?

```suggestion
    wahlbezirkid       varchar(1000) not null,
    eroeffnungsuhrzeit TIMESTAMP     not null,
    primary key (wahlbezirkid)
```

Reply:
Noch einmal die Antwort, nein hat er nicht.
Noch einmal weil, ich weiß es nicht was Chrome im Moment hat, alle Bereiche mit suggested change erscheinen doppelt